Tools/wpe/backends/fdo/WindowViewBackend.cpp

```diff
@@ -1063,7 +1063,20 @@ void WindowViewBackend::displayBuffer(struct wpe_fdo_egl_exported_image* image)
     struct wl_callback* callback = wl_surface_frame(m_surface);
     wl_callback_add_listener(callback, &s_frameListener, this);
 
+#if defined(ENABLE_BUFFER_DAMAGE_TRACKING) && ENABLE_BUFFER_DAMAGE_TRACKING
+    const int32_t* damageRegions;
+    auto damageRegionsCount = wpe_fdo_egl_exported_image_get_damage_regions(image, &damageRegions);
+    if (damageRegionsCount && damageRegions) {
+        // FIXME Ubuntu's libepoxy seems to have pulled an older API version where the rects param isn't const
+        // FIXME Is there any case where we should call EXT instead of KHR?
```

Comment:
Pick one of them, libepoxy will choose one or the other automatically depending on driver capabilities; as they are both marked as aliases of each other 😉